securedrop_client/api_jobs/sync.py

```diff
@@ -40,6 +41,16 @@ def call_api(self, api_client: API, session: Session) -> Any:
         sources, submissions, replies = get_remote_data(api_client)
 
         update_local_storage(session, sources, submissions, replies, self.data_dir)
+
+        for source in sources:
+            source_id = source.uuid
+            files = []
+            for submission in submissions:
+                if submission.source_id == source_id and submission.is_file():
+                    files.append(submission.uuid)
+            conversation_id = source_id
```

Comment:
I think it makes sense to set the `conversation_id` to the `source_id` when the `ConversationView` is created, or when the `ConversationState` is created.